src/main.js

```diff
@@ -8,9 +8,11 @@ import BootstrapVue from "bootstrap-vue";
 import moment from "moment";
 import "bootstrap/dist/css/bootstrap.css";
 import "bootstrap-vue/dist/bootstrap-vue.css";
+import { Mediaelement } from "vue-mediaelement/src";
 
 Vue.use(VuePlugin);
 Vue.use(BootstrapVue);
+Vue.component("mediaelement", Mediaelement);
 
 Vue.use(VueFilterDateFormat, {
   dayOfWeekNames: [

```

Comment:
Is main.js the best place to register Mediaelement? see [Mediaelement](https://openbase.com/js/vue-mediaelement)